blocks/init/src/blocks/components/menu-drawer/menu-drawer.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * Mobile menu as drawer
+ *
+ * @package Eightshift_Boilerplate\Components
+ *
+ * @since 1.0.0
+ */
+
+use Eightshift_Libs\Menu\Menu;
+use Eightshift_Libs\Blocks\Helpers\Components;
+
+$block_class     = $attributes['blockClass'] ?? 'menu-drawer';
+$menu            = $attributes['menu'] ?? 'header_main_nav';
+$drawer_position = $attributes['drawerPosition'] ?? 'left';
+
+$classes = Components::classnames([
+  $block_class,
+  "js-{$block_class}",
+  "{$block_class}--{$drawer_position}",
+]);
+
+?>
+<div class="<?php echo esc_attr( $classes ); ?>">
+  <?php echo esc_html( Menu::bem_menu( $menu, 'menu-drawer-nav', '' ) ); ?>
```

Comment:
if you have a standalone class for menu are you duplicating this component here also?

Reply:
Related to https://github.com/infinum/eightshift-frontend-libs/pull/83#discussion_r380016180, we should probably discuss how this can be done better IRL.